### PR TITLE
Fix type errors on ember canary

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
This should hopefully improve the ember-beta and ember-canary CI

### CODE REVIEW

File: packages/@glimmer/component/package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-qunit": "^4.6.0",
     "ember-resolver": "^5.0.0",
-    "ember-source": "~3.16.0",
+    "ember-source": "https://s3.amazonaws.com/builds.emberjs.com/beta/shas/60d96396ff8b31212b742e38b77ee103917d9d8b.tgz",
```
Comment (NullVoxPopuli) [inline packages/@glimmer/component/package.json:77]: whoops

File: packages/@glimmer/component/addon/-private/base-component-manager.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { DEBUG } from '@glimmer/env';
-import { ComponentManager, ComponentCapabilities, TemplateArgs } from '@glimmer/core';
```
Comment (NullVoxPopuli) [inline packages/@glimmer/component/addon/-private/base-component-manager.ts:2]: TemplateArgs doesn't exist

Comment (NullVoxPopuli) [reply packages/@glimmer/component/addon/-private/base-component-manager.ts:2]:
```
addon/-private/base-component-manager.ts:2:51 - error TS2305: Module '"../../../core/dist/commonjs"' has no exported member 'TemplateArgs'.

2 import { ComponentManager, ComponentCapabilities, TemplateArgs } from '@glimmer/core';
                                                    ~~~~~~~~~~~~
```

**Comment by NullVoxPopuli** [comment]
Blocked by https://github.com/glimmerjs/glimmer.js/pull/361

File: packages/@glimmer/component/addon/-private/base-component-manager.ts (hunk 2 above)
Comment (rwjblue) [reply packages/@glimmer/component/addon/-private/base-component-manager.ts:2]: Take a look at https://github.com/glimmerjs/glimmer.js/pull/334, looks like this was just missed.

**Pull request merged**